boards/cats_rev1Pro/Core/Inc/config/globals.h

```diff
@@ -49,7 +49,8 @@ extern fifo_t usb_output_fifo;
 extern uint8_t usb_fifo_out_buffer[256];
 
 /** State Estimation **/
-
+/* Index of 100g Accel */
+#define ACC_100_INDEX 2
```

Comment:
Actually the accel goes up to 200g on this version. Maybe instead of having the value we could call it `HIGH_G`

Reply:
Will fix